### PR TITLE
Added the linux sections to connecting to the cluster

### PR DESCRIPTION
**Pull request by joseph-cooke**

### CODE REVIEW

**Review by trudenkoNE — CHANGES_REQUESTED** [review]
⋐(◉▾◉)⋑
[EEEEEEE]

Owl of approval types 'they're all step one'

**Review by SerenaCaplins — COMMENTED** [review]
looks good to me. Step numbers render correctly in read the docs.

However I think we eventually want to burn down "first steps" So should we be moving the passwordless ssh to within each connecting to cluster section?

**Review by SerenaCaplins — APPROVED** [review]
looks good. First steps is ready to burn down

**Comment by joseph-cooke** [comment]
First steps is burned down. bashrc content is now in best practices and conda bashrc items are in the conda page.

File: docs/source/best-practices/shell_environment.md [hunk 1]
```diff
@@ -1,9 +1,7 @@
-(shell-environment-on-cluster)=
-
+(shell_environment)=
 # Shell Environment on the Cluster
 
 ## The Discovery Shell environment and `.bashrc`
```
Comment (tketchem14) [inline docs/source/best-practices/shell_environment.md:4]: Please capitalize Environment. 
```suggestion
## The Discovery Shell Environment and `.bashrc`
```

**Review by Rakshith2605 — CHANGES_REQUESTED** [review]
"Update email link formatting in Markdown to hide 'mailto' prefix"

File: docs/source/best-practices/shell_environment.md [hunk 2]
```diff
@@ -1,46 +1,38 @@
...
-
 Discovery uses a Linux-based Operating System (CentOS), where the Shell program interfaces with the user. Bash (Bourne Again SHell) is one of the most popular Shell implementations, the default Shell on Discovery.
 
 The Shell script `.bashrc` is used by Bash to initialize your Shell environment. For example, it is typically used to define aliases, functions, and load modules. Note that environment variables settings (such as `PATH`) generally go in the `.bash_profile` or `.profile` files. Your `.bashrc`, `.bash_profile`, and `.profile` files live in your `$HOME` directory. You can change your .bashrc with a text editor like [nano](https://www.nano-editor.org/).
 
 :::{caution}
 Making edits to your `.bashrc` file can result in many issues. Some changes may prevent you from launching apps or executing commands. Modifying your `PATH` variable may result in the inability to use basic Shell commands (such as `cd` or `ls`) if not done correctly.
```
Comment (Rakshith2605) [inline docs/source/best-practices/shell_environment.md:10]:
```suggestion
Making edits to your `.bashrc` file can result in many issues. Some changes may prevent you from launching apps or executing commands. Modifying your `PATH` variable may result in the inability to use basic shell commands (such as `cd` or `ls`) if not done correctly.
```

File: docs/source/best-practices/shell_environment.md [hunk 3]
```diff
@@ -1,46 +1,38 @@
-(shell-environment-on-cluster)=
-
+(shell_environment)=
 # Shell Environment on the Cluster
 
 ## The Discovery Shell environment and `.bashrc`
-
 Discovery uses a Linux-based Operating System (CentOS), where the Shell program interfaces with the user. Bash (Bourne Again SHell) is one of the most popular Shell implementations, the default Shell on Discovery.
 
 The Shell script `.bashrc` is used by Bash to initialize your Shell environment. For example, it is typically used to define aliases, functions, and load modules. Note that environment variables settings (such as `PATH`) generally go in the `.bash_profile` or `.profile` files. Your `.bashrc`, `.bash_profile`, and `.profile` files live in your `$HOME` directory. You can change your .bashrc with a text editor like [nano](https://www.nano-editor.org/).
```
Comment (Rakshith2605) [inline docs/source/best-practices/shell_environment.md:7]:
```suggestion
The shell script `.bashrc` is used by `bash` to initialize your Shell environment. For example, it is typically used to define aliases, functions, and load modules. Note that environment variables settings (such as `PATH`) generally go in the `.bash_profile` or `.profile` files. Your `.bashrc`, `.bash_profile`, and `.profile` files live in your `$HOME` directory. You can change your .bashrc with a text editor like [nano](https://www.nano-editor.org/).
```

File: docs/source/best-practices/shell_environment.md [hunk 4]
```diff
@@ -1,46 +1,38 @@
-(shell-environment-on-cluster)=
-
+(shell_environment)=
 # Shell Environment on the Cluster
 
 ## The Discovery Shell environment and `.bashrc`
-
 Discovery uses a Linux-based Operating System (CentOS), where the Shell program interfaces with the user. Bash (Bourne Again SHell) is one of the most popular Shell implementations, the default Shell on Discovery.
```
Comment (Rakshith2605) [inline docs/source/best-practices/shell_environment.md:5]:
```suggestion
Discovery uses a Linux-based operating system (CentOS), where the shell program interfaces with the user. Bash (Bourne Again SHell) is one of the most popular shell implementations, the default shell on Discovery.
```

**Review by Rakshith2605 — CHANGES_REQUESTED** [review]
"Update email link formatting in Markdown to hide 'mailto' prefix"

**Review by joseph-cooke — COMMENTED** [review]
The package auto converts the numbered lists into a properly numbered list if 1. is just used so it renders properly. It took me a second to see it also because the source looks differently. (https://northeastern-university-rc-public-documentation--237.com.readthedocs.build/en/237/gettingstarted/connectingtocluster/linux.html)

**Review by trudenkoNE — APPROVED** [review]
⋐(◉▾◉)⋑

lgtm

The systemctl bit in shell_environment imo isn't something that the users would really be using. I think we could save a few lines there.

File: docs/source/best-practices/shell_environment.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+:::{code} bash
+# .bashrc
+
+# Source global definitions
+if [ -f /etc/bashrc ]; then
+	. /etc/bashrc
+fi
+
+# Uncomment the following line if you don't like systemctl's auto-paging feature:
```
Comment (Rakshith2605) [inline docs/source/best-practices/shell_environment.md:26]:
```suggestion
# Uncomment the following line if you don't like SystemCTL's auto-paging feature:
```

File: docs/source/gettingstarted/connectingtocluster/index.md [hunk 6]
```diff
@@ -5,38 +5,51 @@
 
 Mac <mac>
 Windows <windows>
+Linux <linux>
 ```
 
-The following sections on this page will guide you in how to connect to the cluster using the command line access using the terminal and the web interface of Open OnDemand. Guides are provided for if your personal computer is running Windows or macOS.
+The following sections on this page will guide you on how to connect to the cluster using the command line access using the terminal and the web interface of Open OnDemand. Guides are provided if your personal computer is running Windows, macOS, or Linux.
```
Comment (Rakshith2605) [inline docs/source/gettingstarted/connectingtocluster/index.md:11]:
```suggestion
The following sections on this page will guide you on how to connect to the cluster using command-line access using the terminal and the web interface of Open OnDemand. Guides are provided if your personal computer is running Windows, macOS, or Linux.
```

**Review by Rakshith2605 — COMMENTED** [review]
Everything looks good.

**Review by Rakshith2605 — APPROVED** [review]

**Pull request merged**